### ISSUE TITLE
Accuracy scorer for BERT-based classifier

### ISSUE THREAD

**Issue by asmithh** [davidatbu/openFraming#28]
Given a validation set and a fine-tuned BERT-based (binary) classifier, return the following:
{
  "macro_f1": float,
  "macro_precision": float,
  "macro_recall": float,
  "accuracy": float
}

May need to do this for each category if we're doing one binary classifier per category.
